src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -59,30 +69,42 @@ const ProfileSelector = ({ input, showSelector }) => {
   };
 
   const handleClear = () => {
-    selectProfile(undefined);
+    setProfile(undefined);
     change(input.name, undefined);
     reinitDependingSteps(change);
   };
 
-  const setProfile = (_, selection) => {
-    selectProfile(selection);
+  const handleSelect = (_, selection) => {
+    setProfile(selection);
     setIsOpen(false);
     change(input.name, selection);
     reinitDependingSteps(change);
     change('file-system-config-radio', 'manual');
   };
 
   return (
-    <FormGroup
-      isRequired={true}
-      label={'Profile to use for this image'}
-      data-testid="profiles-form-group"
-    >
+    <FormGroup isRequired={true} data-testid="profiles-form-group">
+      {'OpenSCAP profile'}
+      <Popover
+        maxWidth="30rem"
+        position="left"
+        bodyContent={
+          <TextContent>
+            <Text>
+              To run a manual compliance scan in OpenSCAP, download this image.
+            </Text>
+          </TextContent>
+        }
+      >
+        <Button variant="plain" aria-label="About OpenSCAP">
+          <HelpIcon />
+        </Button>
+      </Popover>
       <Select
```

Comment:
Should the select work as a typeahead select? Typing into it doesn't seem to filter the results. Also when you type something and hit enter to possible trigger the filtering the whole wizard jumps back to the first step.